doc/api/cli.md

```diff
@@ -240,6 +240,13 @@ resolving relative paths.
 
 See `--preserve-symlinks` for more information.
 
+### `--prof`
+<!-- YAML
+added: v5.2.0
```

Comment:
Correct...? 🤔

Reply:
I don't know the version it was introduced in, but it is much older than that. The flag already exists in 0.10.

Reply:
I just checked and the log file is generated from v.2.0.0 on.